src/modules/infobox/components/FeatureProfile.vue

```diff
@@ -79,7 +79,7 @@ export default {
         return {
             showTooltip: false,
             options: {
-                margin: { left: 35, right: 15, bottom: 25, top: 15 },
```

Comment:
Is that wanted? That's something you changed back with https://github.com/geoadmin/web-mapviewer/pull/215

Reply:
Yes. On #215 I needed bigger margins on the right and the top but wanted to keep overall size so I reduced left and bottom. But on this commit I see that I actually need this big margins also on the left and bottom to display the x and y labels.